bugbot/rules/perfalert_resolved_regression.py

```diff
@@ -0,0 +1,200 @@
+            if not bug_info.get("resolution"):
+                for specific_change in change["changes"]:
+                    if specific_change["field_name"] == "resolution":
+                        bug_info["resolution"] = specific_change["added"]
+                        bug_info["resolution_previous"] = (
+                            specific_change["removed"].strip() or "---"
+                        )
+                        bug_info["resolution_time"] = change["when"]
+                        break
+
+            if bug_info.get("resolution"):
+                # Find the status that the bug was resolved to, and by who
+                for specific_change in change["changes"]:
+                    if specific_change["field_name"] == "status" and specific_change[
+                        "added"
+                    ] in ("RESOLVED", "REOPENED"):
+                        bug_info["status"] = specific_change["added"]
+                        bug_info["status_author"] = change["who"]
+                        bug_info["status_time"] = change["when"]
+                        break
+
+            if bug_info.get("status"):
+                break
+
+        return bug_info
+
+    def set_autofix(self, bugs):
+        for bug_id, bug_info in bugs.items():
+            if bug_info["needinfo"]:
+                self.extra_ni[bug_id] = {
+                    "resolution": bug_info["resolution"],
+                    "status": bug_info["status"],
+                }
+                self.add_auto_ni(
+                    bug_id,
+                    {
+                        "mail": bug_info["status_author"],
+                        "nickname": bug_info["nickname"],
+                    },
+                )
+
+    def get_needinfo_nicks(self, bugs):
+        def _user_handler(user, data):
+            data[user["name"]] = user["nick"]
+
+        authors_to_ni = set()
+        for bug_id, bug_info in bugs.items():
+            if bug_info["needinfo"]:
+                authors_to_ni.add(bug_info["status_author"])
+
+        if not authors_to_ni:
+            return
+
+        user_emails_to_names = {}
+        BugzillaUser(
+            user_names=list(authors_to_ni),
+            include_fields=["nick", "name"],
+            user_handler=_user_handler,
+            user_data=user_emails_to_names,
+        ).wait()
+
+        for bug_id, bug_info in bugs.items():
+            if bug_info["needinfo"]:
+                bug_info["nickname"] = user_emails_to_names[bug_info["status_author"]]
+
+    def handle_bug(self, bug, data):
+        # Match all the resolutions with resolution comments if they exist
+        bug_id = str(bug["id"])
+        bug_comments = bug["comments"]
+        bug_history = self.get_resolution_history(bug)
+
+        # Sometimes a resolution comment is not provided so use a default
+        bug_history["needinfo"] = False
+        bug_history["resolution_comment"] = DEFAULT_RESOLUTION_COMMENT
```

Comment:
What do you think about using `n/a` instead? It will be shorter in the email and easier to spot.